ftw/linkchecker/configuration.py

```diff
@@ -22,7 +22,7 @@ def _parse_arguments(self, args):
                             help="Path to linkchecker config file.")
         parser.add_argument("--log",
                             help="Path to linkchecker log file.")
-        args, unknown = parser.parse_known_args()
+        args, unknown = parser.parse_args()
```

Comment:
I suspect you mean this: :-)

```suggestion
        args = parser.parse_args()
```

Reply:
Oh yes! I totally missed that. Thanks a lot.